### ISSUE TITLE
implement mkfifo()

### ISSUE THREAD

**Issue by anton-bondarev** [embox/embox#1818]
mkfifo() has not been implemented yet

**Comment by zhuanyanchengkong** [comment]
> mkfifo() has not been implemented yet

Hi, @anton-bondarev Let me try to solve this problem.

**Comment by anton-bondarev** [comment]
@zhuanyanchengkong I've assigned it with you.
But I think it's rather difficult issue!

**Comment by anton-bondarev** [comment]
@zhuanyanchengkong Please, let finish #1806 first!

**Comment by zhuanyanchengkong** [comment]
@anton-bondarev Hi,I am using mknod to complete the mkfifo function, do you have any suggestions.

**Comment by anton-bondarev** [comment]
@zhuanyanchengkong You can try use mknod, of cause, but except inode creation you have to assign it with fifo functionality.
I think It's better to use fs routines directly.
inode_new(), node_alloc() etc.

**Comment by zhuanyanchengkong** [comment]
@anton-bondarev  Yes, I also found that mknod command in embox is incomplete. I want to complete the `vfs_mknod ()` function, and if the embox needs to complete the mknod command, it can also be used directly.

**Comment by zhuanyanchengkong** [comment]
@anton-bondarev Hi, I have some problems. The current mknod command cannot set the device number, so do I only need to set the mode in `child = __vfs_subtree_create_child(*tmp_parent, path, len, mode);` to S_IFIFO to use the current mknod code to implement mkfifo?
